### PR TITLE
BETA: Register fmode.is-a.dev

### PR DESCRIPTION
**Pull request by jOaawd**
Added `fmode.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/fmode.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "jOaawd",
+        "email": "jorti28@apps.pusd.org"
+    },
+    "record": {
+        "CNAME": "https://goblocked.github.io/"
```
Comment (CuteDog5695) [inline domains/fmode.json:7]: you can't have https:// and / in CNAME record

File: domains/googleclassroom.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "jOaawd",
+        "email": "jorti28@apps.pusd.org"
+    },
+    "record": {
+        "TXT": "google-site-verification=i3wpnm7r-xxn3ct-dilpt6ka1ylop-xo53gwl8wtet0"
```
Comment (CuteDog5695) [inline domains/googleclassroom.json:7]: we dont support google site txt verification

File: domains/theublocked.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "jOaawd",
+        "email": "jorti28@apps.pusd.org"
+    },
+    "record": {
+        "TXT": "google-site-verification=6gmt5zyihimbubddc8hei7ysebshvsvotvt0rpfhk24"
```
Comment (CuteDog5695) [inline domains/theublocked.json:7]: we dont support google site txt verification

File: domains/classroom.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "jOaawd",
+        "email": "jorti28@apps.pusd.org"
+    },
+    "record": {
+        "CNAME": "gv-pfibmgoyouk6yz.dv.googlehosted.com"
```
Comment (CuteDog5695) [inline domains/classroom.json:7]: site cant be reached

File: domains/home.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "jOaawd",
+        "email": "jorti28@apps.pusd.org"
+    },
+    "record": {
+        "URL": "https://roblox.com"
```
Comment (CuteDog5695) [inline domains/home.json:7]: can't you just redirect site to your website instead of roblox?

**Review by iostpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**